src/app/clusters/media-playback-server/media-playback-server.cpp

```diff
@@ -39,74 +39,158 @@
  ******************************************************************************/
 
 #include <app/Command.h>
+#include <app/clusters/media-playback-server/media-playback-server.h>
+#include <app/common/gen/attribute-id.h>
+#include <app/common/gen/attribute-type.h>
+#include <app/common/gen/cluster-id.h>
+#include <app/common/gen/command-id.h>
+#include <app/common/gen/enums.h>
 #include <app/util/af.h>
 
-bool emberAfMediaPlaybackClusterFastForwardRequestCallback(chip::app::Command * commandObj)
+EmberAfMediaPlaybackStatus mediaPlaybackClusterSendMediaPlaybackRequest(MediaPlaybackRequest mediaPlaybackRequest,
+                                                                        uint64_t deltaPositionMilliseconds);
+
+uint8_t mediaPlaybackClusterPlaybackState;
```

Comment:
Why do we need this global variable?  The one function it's used in could easily use a function-local.

Reply:
Makes sense, moved it to function.